aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -40,6 +40,7 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.ristretto255.basepoint_double_mul, "ristretto255.basepoint_double_mul", 300],
 
     [.ristretto255.point_add, "ristretto255.point_add", 300],
+    [.ristretto255.point_clone, "ristretto255.point_clone", 300],
```

Comment:
You need to mark the new parameters optional

Reply:
@wrwg previously said that, with feature gating, marking gas parameters as optional is no longer required?

Reply:
I assume I still need to mark these as optional.